### DISCUSSION TITLE
BMP to TIFF without additional metadata

### DISCUSSION THREAD

**Discussion by restaurierung-digital** [ImageMagick/ImageMagick#3223] [Help]
I want to convert BMP to TIFF.

I use `magick convert "file.bmp" "file.tif"` 
Edit: or `magick "file.bmp" "file.tif"`

This creates the following additional metadata:
IFD0:PageNumber
IFD0:PrimaryChromaticities
IFD0:WhitePoint

Since this information was not present in the original file, it should not be in the converted file. I know that I can delete the metadata with e.g. ExifTool. But how can I convert the files directly with Imagemagick so that no additional metadata is created?

Thanks

**Comment by urban-warrior** [comment]
Add `-strip` to your command line.

**Reply by restaurierung-digital** [reply]
Thanks, but 
`magick example.bmp -strip example.tif` 
does not change anything. The mentioned metadata will still be added.

Version: ImageMagick 7.0.11-0 Q16 x64 2021-02-13
